### PR TITLE
Release 0.47.0

### PR DESCRIPTION
**Pull request by ishustava**
Changes proposed in this PR:
- Update versions for the release
- Downgrade default envoy version because 1.23.0 doesn't work on linux arm64: https://github.com/envoyproxy/envoy/issues/22261

How I've tested this PR:

How I expect reviewers to test this PR:

Checklist:
- [ ] Tests added
- [ ] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## UNRELEASED
+## 0.47.0 (August 11, 2022)
```
Comment (ishustava) [inline CHANGELOG.md:1]: need to change this if we are going to release tomorrow instead of today

**Review by curtbushko — APPROVED** [review]
Looks good!

**Review by thisisnotashwin — APPROVED** [review]
🙏

**Review by thisisnotashwin — COMMENTED** [review]
Should we update the version of the image in the circle ci and github actions config as well?
Also, I think we can update the API and SDK versions to 1.14.0 and 0.11.0

**Pull request merged**